docs/source/changelog.md

```diff
@@ -6,6 +6,61 @@ command line for details.
 
 ## [Unreleased]
 
+## 13
+
+### 13.0 2023-11
```

Comment:
A link update in the end of the file is needed for consistency here.

I think having the exact date is very helpful as well because I often track down _why did this break now?_ and look for changes between specific dates in dependencies. This is not what we have done historically in this file though.

Reply:
Good point! It's only the 12.* releases that are missing the full date, [earlier releases have them](https://github.com/jupyterhub/dockerspawner/blob/main/docs/source/changelog.md#011)

Reply:
If you want to fill them in now the [tag dates](https://github.com/jupyterhub/dockerspawner/tags) are
12.1.0: 2021-07-22
12.0.0: 2021-03-26